frontend/src/containers/ViewDashboardAdmin.tsx

```diff
@@ -163,25 +163,33 @@ function ViewDashboardAdmin() {
       <Modal
         isOpen={isOpenUpdateModal}
         closeModal={() => setIsOpenUpdateModal(false)}
-        title={t("CreateDraftDasboardModalTitle", { name: dashboard?.name })}
-        message={t("CreateDraftDasboardModalMessage")}
-        buttonType={t("CreateDraftDasboardModalButton")}
+        title={`${t("CreateDraftDashboardModalTitlePrefix")} ${
+          dashboard?.name
+        } ${t("CreateDraftDashboardModalTitleSuffix")}`}
+        message={t("CreateDraftDashboardModalMessage")}
+        buttonType={t("CreateDraftDashboardModalButton")}
         buttonAction={onUpdateDashboard}
       />
 
       <Modal
         isOpen={isOpenArchiveModal}
         closeModal={() => setIsOpenArchiveModal(false)}
-        title={t("ArchiveDashboardModalTitle", { name: dashboard?.name })}
-        message={t("ArchiveDashboardModalMessage", { name: dashboard?.name })}
+        title={`${t("ArchiveDashboardModalTitlePrefix")} ${dashboard?.name} ${t(
+          "ArchiveDashboardModalTitleSuffix"
+        )}`}
+        message={`${t("ArchiveDashboardModalMessagePrefix")} ${
+          dashboard?.name
+        } ${t("ArchiveDashboardModalMessageSuffix")}`}
         buttonType={t("ArchiveDashboardModalButton")}
         buttonAction={onArchiveDashboard}
       />
 
       <Modal
         isOpen={isOpenRepublishModal}
         closeModal={() => setIsOpenRepublishModal(false)}
-        title={t("RepublishDashboardModalTitle", { name: dashboard?.name })}
+        title={`${t("RepublishDashboardModalTitlePrefix")} ${
```

Comment:
@yinxudeng instead of "suffix" and "prefix", how about using a hierarchy like
PreparePublishingModalTitle:{
"part1":"something",
"part2":"something else"
}
We do this in other places, and the translator can understand that the texts are related